scipy-1.0/paper.tex

```diff
@@ -791,12 +791,17 @@ \subsubsection*{Test suite}
 are more than \num{13000}~unit tests in the test suite, which is written for usage
 with the \texttt{pytest}\cite{pytest} framework.
 Test coverage at the SciPy~1.0 release point (Figure~\ref{fig:coverage}) was at
-87\% for Python code according to \texttt{pytest-cov}\cite{pytest-cov} and
-45\% for compiled (C, C++, and Fortran) code according to \texttt{gcov}\cite{gcov}.
+87\% for Python code according to \texttt{pytest-cov}\cite{pytest-cov}.
+Coverage of compiled (C, C++, and Fortran) code was only 45\% according to
+\texttt{gcov}\cite{gcov}, but the compiled codebase is much more robust than
+this figure would suggest as the figure does not correct for the inclusion of
+reputable vendored code, the original library of which is well-tested;
+generated code, for which full coverage is impractical; and
+deprecated code, which does not require unit tests.
 Documentation for the code is automatically built and published by
 the CircleCI service to facilitate evaluation of documentation changes /
-integrity.  Our full test suite also passes with PyPy3\cite{Bolz:2009:TMP:1565824.1565827}, a just-in-time compiled
-version of the Python language.
+integrity. % Our full test suite also passes with PyPy3\cite{Bolz:2009:TMP:1565824.1565827},
```

Comment:
This is now mentioned elsewhere. I'm not sure if it's worth repeating and again qualifying that it is only as of the 1.0 release point.

Reply:
Sensible -- marking as resolved